src/Test.UnitTests.Sarif.Multitool/PageCommandTests.cs

```diff
@@ -8,7 +8,7 @@
 using Newtonsoft.Json;
 using Xunit;
 
-namespace Sarif.Multitool.UnitTests
+namespace Microsoft.CodeAnalysis.Test.UnitTests.Sarif.Multitool
```

Comment:
>Multitool [](start = 54, length = 9)

Yay!

Reply:
i knew you'd like this!

---
In reply to: [298666308](https://github.com/microsoft/sarif-sdk/pull/1549#discussion_r298666308) [](ancestors = 298666308)